package.json

```diff
@@ -99,7 +99,8 @@
     "p-lazy": "^1.0.0",
     "prettier": "^1.5.3",
     "resolve-cwd": "^2.0.0",
-    "supports-color": "^5.2.0",
+    "supports-color": "^5.3.0",
+    "uglifyjs-webpack-plugin": "^1.2.2",
```

Comment:
Is this new? :O

Reply:
Ah! Almost nobody noticed it. :laughing: :eyes:

Reply:
We should remove it

Reply:
Woops, a bad rebase on my end. It got removed in #309 and I merged badly. Will remove.